### PR TITLE
IDEMPIERE-5393 - Allow rendering status line as widget/gadget in dashboard

### PR DESCRIPTION
**Pull request by PeterTakacs300**
JIRA ticket: https://idempiere.atlassian.net/browse/IDEMPIERE-5393

### CODE REVIEW

File: org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java [hunk 1]
```diff
@@ -559,6 +560,19 @@ public void onEvent(AfterSizeEvent event) throws Exception {
...
     	
+    	// Status Line
+    	final int AD_StatusLine_ID = dc.getAD_StatusLine_ID();
+    	if(AD_StatusLine_ID > 0) {
+    		MStatusLine sl = new MStatusLine(Env.getCtx(), AD_StatusLine_ID, null);
+    		final Html statusLineHtml = new Html();
+    		statusLineHtml.setContent(sl.parseLine(0));
+    		Div div = new Div();
+    		div.appendChild(statusLineHtml);
+    		div.setStyle("padding: 0px 10px 10px;");
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/desktop/DashboardController.java:571]: Should use css instead, for e.g a new css class of "statusline-gadget".

File: org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/gadget.css.dsp [hunk 2]
```diff
@@ -212,6 +212,10 @@
 	height: 100% !important; 
 }
 
+.statusline-gadget {
+	padding: 0px 10px 10px;
+}
+
 .help-content
```
Comment (hengsin) [inline org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/gadget.css.dsp:219]: please update the "idempiere.theme.default" version in org.adempiere.ui.zk/WEB-INF/src/metainfo/zk/lang-addon.xml

**Review by hengsin — CHANGES_REQUESTED** [review]

File: org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/gadget.css.dsp (hunk 2 above)
Comment (PeterTakacs300) [reply org.adempiere.ui.zk/WEB-INF/src/web/theme/default/css/fragment/gadget.css.dsp:219]: Hi @hengsin, fixed

**Review by hengsin — APPROVED** [review]
tested

**Pull request merged**